documentation/source/references.bib

```diff
@@ -60,3 +60,11 @@ @misc{chen2023randomly
   archiveprefix = {arXiv},
   primaryclass  = {math.NA}
 }
+
+@inproceedings{benard2023kernel,
+  title     = {Kernel {S}tein Discrepancy thinning: a theoretical perspective of pathologies and a practical fix with regularization},
```

Comment:
Although bibtex does suggest the style `{S}tein` for capitalization, to avoid confusing the spell checker, we can instead do the below with no loss of fuctionality (we still preserve the capitalization).
```suggestion
  title     = {Kernel {Stein} Discrepancy thinning: a theoretical perspective of pathologies and a practical fix with regularization},
```

Reply:
Done.